openmm_md/plain_md.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    settings.simulation_settings.equilibration_length_nvt = 0.01 * unit.nanosecond
</pre><p> </p><p>[enhancement]</p><p> </p><p>To help the reader, it may be worth adding inline comments here about what these are doing?</p><p> </p><p>Something along the lines of "# setting the nvt equilibration length to 0.1 ps"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Added this!</p>